microcosm_flask/tests/conventions/test_command.py

```diff
@@ -0,0 +1,130 @@
+"""
+Command operation tests.
+
+"""
+from json import dumps, loads
+from hamcrest import (
+    assert_that,
+    equal_to,
+    is_,
+)
+
+from marshmallow import fields, Schema
+from microcosm.api import create_object_graph
+
+from microcosm_flask.conventions.encoding import dump_response_data, load_request_data
+from microcosm_flask.conventions.registry import request, response
+from microcosm_flask.operations import Operation
+
+
+class CommandArgumentSchema(Schema):
+    value = fields.String(required=True)
+
+
+class CommandResultSchema(Schema):
+    result = fields.Boolean(required=True)
+    value = fields.String(required=True)
+
+
+def make_command(graph, request_schema, response_schema):
+    """
+    Create an example command route.
```

Comment:
I'm with you. These endpoints are a nod to practicality, not "good" design. I also wouldn't use the term "virtual" here; I think that implies (incorrectly) that HTTP resources need to have a one-to-one mapping in the persistence layer.

This feature came about because we were discussing how to migrate some of the existing endpoints that are not RESTful. It's going to be less error prone to migrate them with their current paths than to reinterpret them in terms of new resources.